cli.js

```diff
@@ -22,40 +22,75 @@ var meow = require('meow');
 var path = require('path');
 var swPrecache = require('./');
 
-function setDefaults(cli) {
-  cli.flags.root = cli.flags.root || './';
-  if (cli.flags.root.lastIndexOf('/') !== cli.flags.root.length - 1) {
-    cli.flags.root += '/';
+function setDefaults(cli, configFileFlags) {
+  var compositeFlags = cli.flags;
```

Comment:
We still officially [support `node >= 0.1.0`](https://github.com/GoogleChrome/sw-precache/blob/master/package.json#L20) as a runtime.

I'd rather rewrite all this code and then run it through Babel and publish that than do _ad hoc_ polyfilling. I can track that in a different bug so as not to delay this.